README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
Hey @jrafanie AFAIK this is not true. I can see where in the CLI this is used. Have you seen otherwise?